docs/user/guides/troubleshooting/index.rst

```diff
@@ -0,0 +1,14 @@
+How to troubleshoot specific issues
+-----------------------------------
+
+In the following guides,
+you can learn how to fix specific common problems.
+
+.. File locations are a bit odd here because of legacy.
+.. We will move files to new locations and create redirects in a later refactor
+
+.. toctree::
+   :maxdepth: 1
+
+   Build errors <../build-troubleshooting>
+   Slow builds <../build-using-too-many-resources>
```

Comment:
I left these pages in their original location intentionally. We will move pages and create redirects in a separate workflow.

Reply:
This is another place where I think it probably makes sense to keep the original titles, since they are so close to these.

Reply:
Are you referring to changing original page titles or the fact that they are explicitly labeled in the TOC/Navigation?

If for the TOC/Navigation, I wanted to have it look like this:

```
Troubleshooting
  Slow builds
  Build errors
```

And not for instance:

```
Troubleshooting guides
  Troubleshooting slow builds
  Troubleshooting build errors
```


WRT page title changes, I wanted to reuse the pattern from before but got distracted...

```

.. _Old Title:

New title
=========

```

Reply:
```
Troubleshooting
  Slow builds
  Build errors
```

Yea.. that looks nicer for sure. I wonder about a feature where the section is included in the h1, as a type of Breadcrumb, so that we don't have to keep overwriting things in TOC's. Probably over-optimizing tho for a smallish use case, and also probably a bunch of places we _don't_ want this :)

Reply:
I want this, too, and I've been wondering if we could write this type of meta-data for pages before and use it in a template-like way. I think it's also okay to write it directly in the TOC. DRY is still retained.. we write the navigation title exactly where it is used, so even though it's a bit disconnected from its original source, it's still in a context where it seems "understandable".

Reply:
We should consider SEO in case we want to automate this somehow. What's better "Troubleshooting: Slow builds" or "Slow builds" in the H1? Depending on the implementation (js/static) this could be a problem or not.

I don't think we are seriously talking about implementing this, tho, but I just want to mention this.

Reply:
Can you clarify? In this case, we would want the full title in the H1 and a shorter version in the navigation.

Reply:
That sounds good to me 👍🏼